code/data_setup/parse_alov_bb.sh

```diff
@@ -1,19 +0,0 @@
-i=0
-while read line
-do
-    files[ $i ]="$line"        
-    (( i++ ))
-done < <(find $1 -type f)
-
-
-echo ${#files[@]}
-echo ${files[0]} | cut -d / -f 3 
-
-for file in "${files[@]}"
-do 
-    sub_dir="$(echo $file | cut -d / -f 3 | cut -d . -f 1),"
```

Comment:
Thanks for the rewrite.  I'll be honest, I don't actually know what `"$(echo $file | cut -d / -f 3 | cut -d . -f 1),"` even does.